caneton/decode.py

```diff
@@ -155,13 +154,14 @@ def message_decode(message_id, message_length, message_data, dbc_json):
         message_binary_length)
 
     if message_info.get('has_multiplexor', False):
-        multiplexing_mode = message_get_current_multiplexing_mode(
+        multiplexor = message_get_multiplexor(
```

Comment:
multiplexor have _endianess_ too (there are just specialized signals). The only information we use in caneton is the value but as we export the function in our API I think it make sense to provide more information and a consistent API with the signal information.